src/Features/Core/Portable/CodeFixes/RoslynAssemblyAttribute.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System;
+
+namespace Microsoft.CodeAnalysis
+{
+    [AttributeUsage(AttributeTargets.Assembly)]
+    internal class RoslynAssemblyAttribute : Attribute { }
```

Comment:
*Personally*, I'd rather we attribute all of our fixers (with a diagnostic to force this) or use some larger mechanism (like msft public key).